platform/security/sts/security-sts-server/src/main/java/ddf/security/sts/StaticStsProperties.java

```diff
@@ -183,6 +183,7 @@ public void configureProperties() throws STSException {
     WSSConfig.init();
   }
 
+  @SuppressWarnings("squid:S2177" /* getResourceManager in parent class is private */)
```

Comment:
✏️ It would be much preferable to simply rename this private method so it does not look like an incorrect override.

Reply:
What would be a good rename? I'm used to getters and setters having a specific naming format.

Reply:
As it's private, it doesn't matter too much. Something like `getResourceManagerExtension()` would work okay.